Assets/Scripts/Collectables/HealthBasedDropper.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Actor;
+using UnityEngine;
+
+namespace Collectables
+{
+    /// <summary>
+    /// A dropper that drops drops based on the health of the entity it's bound to.
```

Comment:
Can we make this a bit more concise?

`Class that drops pickups based on the health of the entity it's bound to.`

Reply:
This no longer applies with the refactor.